app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
```

Comment:
bear with my little context, that looks like a template html file. can it accept conditions? As in, 

if mapping:
    Mapping: $mapping_var
else:
    <i> none </i>

don't really have a preference. wfm if it disappeared entirely.